mlflow/gateway/providers/anthropic.py

```diff
@@ -36,6 +36,12 @@ async def completions(self, payload: completions.RequestPayload) -> completions.
                 detail="Setting the 'stream' parameter to 'true' is not supported with the MLflow "
                 "Gateway.",
             )
+        candidate_count = payload.get("candidate_count", 1)
+        if candidate_count != 1:
+            raise HTTPException(
+                status_code=422,
+                detail="'candidate_count' must be '1' for the Anthropic provider, if specified.",
+            )
```

Comment:
Could we alert the user of the invalid value that they passed in to be helpful to them? :) 
```suggestion
        if (candidate_count := payload.get("candidate_count", 1)) > 1:
    raise HTTPException(
        status_code=422,
        detail=f"'candidate_count' must not exceed '1' for the Anthropic provider. Current value: {candidate_count}.",
    )

```